aopy/data/bmi3d.py

```diff
@@ -13,6 +13,8 @@
 import h5py
 import tables
 import pandas as pd
+import json
+import sympy
```

Comment:
you will need to add sympy to /docs/source/conf.py under 'mock_imports' as well so that the docs don't get messed up. see https://analyze.readthedocs.io/en/latest/code-guidelines.html?highlight=mock#adding-new-packages